www/index.html

```diff
@@ -1,53 +1,274 @@
-     KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
 -->
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <meta name="format-detection" content="telephone=no" />
-        <!-- WARNING: for iOS 7, remove the width=device-width and height=device-height attributes. See https://issues.apache.org/jira/browse/CB-4323 -->
-        <!-- <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi" /> -->
-        <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1" />
-        <link rel="stylesheet" type="text/css" href="css/index.css" />
-        <title>Connect SDK API Sampler</title>
-        <script type="text/javascript" src="lib/enyo/enyo.js"></script>
-        <link rel="stylesheet" type="text/css" href="lib/enyo/enyo.css" />
-        <script type="text/javascript" src="js/package.js"></script>
-    </head>
-    <body>
-        <script type="text/javascript" src="cordova.js"></script>
-        <script type="text/javascript">
-            function main () {
-                console.log("running main");
-                var app = window.app = new SamplerApp();
-                app.start();
-            }
-            
-            if (window.cordova) {
-                document.addEventListener("deviceready", function () {
-                    main();
-                });
-            } else {
-                enyo.ready(function () {
-                    main();
-                });
-            }
-        </script>
-    </body>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="format-detection" content="telephone=no" />
+		<!-- WARNING: for iOS 7, remove the width=device-width and height=device-height attributes. See https://issues.apache.org/jira/browse/CB-4323 -->
+		<!-- <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi" /> -->
+		<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1" />
+		<link rel="stylesheet" type="text/css" href="css/index.css" />
+		<title>Connect SDK API Sampler</title>
+		<script type="text/javascript" src="lib/enyo/enyo.js"></script>
+		<link rel="stylesheet" type="text/css" href="lib/enyo/enyo.css" />
+		<script type="text/javascript" src="js/package.js"></script>
+	</head>
+	<body>
+		<script type="text/javascript" src="cordova.js"></script>
+		<script type="text/javascript">
```

Comment:
It may be better to extract javascript into external js file and keep it separate from HTML. What do you think?